app/code/Magento/CatalogGraphQl/Model/Resolver/Products/DataProvider/Product.php

```diff
@@ -74,7 +74,7 @@ public function getList(
     ): SearchResultsInterface {
         /** @var \Magento\Catalog\Model\ResourceModel\Product\Collection $collection */
         $collection = $this->collectionFactory->create();
-
+        $attributes = array_unique(array_merge($attributes, ['small_image', 'thumbnail', 'image']));
```

Comment:
Looks like this line is related to another PR.

Reply:
these changes were removed